### DISCUSSION TITLE
ansible-lint not working

### DISCUSSION THREAD

**Discussion by ericcames** [ansible/ansible-lint#2655] [Q&A]
##### Summary

Trying to use ansible-lint on my RHEL 8 laptop and am getting the following error:

ansible-lint --version
WARNING: PATH altered to include /usr/bin
FATAL: Ansible CLI (2.12.2) and python module (2.11.12) versions do not match. This indicates a broken execution environment.

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
ansible --version
```DEPRECATION WARNING]: Ansible will require Python 3.8 or newer on the controller starting with Ansible 2.12. Current version: 3.6.8 (default, Jun 
14 2022, 12:54:58) [GCC 8.5.0 20210514 (Red Hat 8.5.0-10)]. This feature will be removed from ansible-core in version 2.12. Deprecation warnings 
can be disabled by setting deprecation_warnings=False in ansible.cfg.
ansible [core 2.11.12] 
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/eames/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/eames/.local/lib/python3.6/site-packages/ansible
  ansible collection location = /home/eames/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/eames/.local/bin/ansible
  python version = 3.6.8 (default, Jun 14 2022, 12:54:58) [GCC 8.5.0 20210514 (Red Hat 8.5.0-10)]
  jinja version = 2.10.1
  libyaml = True```


ansible-lint --version

```ansible-lint --version
WARNING: PATH altered to include /usr/bin
FATAL: Ansible CLI (2.12.2) and python module (2.11.12) versions do not match. This indicates a broken execution environment.
```

- ansible installation method: one of source, pip, OS package
RPM
- ansible-lint installation method: one of source, pip, OS package
pip
##### OS / ENVIRONMENT

cat /etc/redhat-release 
Red Hat Enterprise Linux release 8.6 (Ootpa)

uname -vr
4.18.0-372.26.1.el8_6.x86_64 #1 SMP Sat Aug 27 02:44:20 EDT 2022

##### STEPS TO REPRODUCE

ansible-lint --version
WARNING: PATH altered to include /usr/bin
FATAL: Ansible CLI (2.12.2) and python module (2.11.12) versions do not match. This indicates a broken execution environment.

```console (paste below)

```

##### Desired Behavior

Possible security bugs should be reported via email to `security@ansible.com`

##### Actual Behavior

ansible-lint -vvvv
WARNING: PATH altered to include /usr/bin
DEBUG    Logging initialized to level 10
DEBUG    Options: Namespace(cache_dir='/home/eames/.cache/ansible-lint/1a54d5', colored=True, config_file=None, configured=True, cwd=PosixPath('/home/eames'), display_relative_path=True, enable_list=[], exclude_paths=['.cache', '.git', '.hg', '.svn', '.tox'], extra_vars=None, format='rich', kinds=[{'jinja2': '**/*.j2'}, {'jinja2': '**/*.j2.*'}, {'inventory': '**/inventory/**.yml'}, {'requirements': '**/meta/requirements.yml'}, {'galaxy': '**/galaxy.yml'}, {'reno': '**/releasenotes/*/*.{yaml,yml}'}, {'playbook': '**/playbooks/*.{yml,yaml}'}, {'playbook': '**/*playbook*.{yml,yaml}'}, {'role': '**/roles/*/'}, {'tasks': '**/tasks/**/*.{yaml,yml}'}, {'handlers': '**/handlers/*.{yaml,yml}'}, {'vars': '**/{host_vars,group_vars,vars,defaults}/**/*.{yaml,yml}'}, {'meta': '**/meta/main.{yaml,yml}'}, {'yaml': '.config/molecule/config.{yaml,yml}'}, {'requirements': '**/molecule/*/{collections,requirements}.{yaml,yml}'}, {'yaml': '**/molecule/*/{base,molecule}.{yaml,yml}'}, {'requirements': '**/requirements.yml'}, {'playbook': '**/molecule/*/*.{yaml,yml}'}, {'yaml': '**/{.ansible-lint,.yamllint}'}, {'yaml': '**/*.{yaml,yml}'}, {'yaml': '**/.*.{yaml,yml}'}], lintables=[], listrules=False, listtags=False, loop_var_prefix=None, mock_modules=[], mock_roles=[], offline=None, parseable=False, parseable_severity=False, progressive=False, project_dir='.', quiet=0, rules={}, rulesdir=[], rulesdirs=['/home/eames/.local/lib/python3.6/site-packages/ansiblelint/rules'], skip_action_validation=True, skip_list=[], tags=[], use_default_rules=False, var_naming_pattern=None, verbosity=4, version=False, warn_list=['experimental', 'role-name'])
DEBUG    /home/eames
ERROR    FATAL: Ansible CLI (2.12.2) and python module (2.11.12) versions do not match. This indicates a broken execution environment.

Please give some details of what is actually happening.
Include a [minimum complete verifiable example] with:

- playbook
- output of running ansible-lint
- if you're getting a stack trace, output of
  `ansible-playbook --syntax-check playbook`

```paste below

```

[minimum complete verifiable example]: http://stackoverflow.com/help/mcve

**Comment by ssbarnea** [comment]
FYI, this was converted to a a discussion because it does not meet any of the criteria required for a bug report.

**Comment by AayushPokharel** [comment]
I too am getting this issue... Any solution on how to solve it?
